src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -1,29 +1,36 @@
 import * as _ from 'lodash'
 import { IsDefined, Min, ValidateIf } from 'class-validator'
 
-import { IsNotBlank } from 'forms/validation/validators/isBlank'
-import { Fractions } from 'forms/validation/validators/fractions'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { Fractions } from 'app/forms/validation/validators/fractions'
+import { MaxLength } from 'app/forms/validation/validators/maxLengthValidator'
 
 export class ValidationErrors {
-  static readonly REASON_REQUIRED: string = 'Enter reason'
+  static readonly REASON_REQUIRED: string = 'Enter a reason'
+  static readonly REASON_TOO_LONG: string = 'Enter a reason no longer than $constraint1 characters'
```

Comment:
Might be worth checking this with Ben, we don't usually say how many characters, just too many characters

Reply:
I will

Reply:
for very long strings we shouldn't tell a number. For shorter strings - we should

Reply:
"You've entered too many characters"